runtime/lua/vim/lsp/completion.lua

```diff
@@ -726,14 +747,21 @@ end
 --- @field autotrigger? boolean  (default: false) When true, completion triggers automatically based on the server's `triggerCharacters`.
 --- @field convert? fun(item: lsp.CompletionItem): table Transforms an LSP CompletionItem to |complete-items|.
 --- @field cmp? fun(a: table, b: table): boolean Comparator for sorting merged completion items from all servers.
+--- @field preselect? boolean moves the item with `lsp.CompletionItem.preselect` to the first. Default: enabled if "noselect" is not present in `completeopt`.
```

Comment:
```suggestion
--- @field preselect? boolean moves the item with `lsp.CompletionItem.preselect` to the first. Default: `false` if "noselect" is present in `completeopt`, `true` otherwise.
```

Feel free to ignore this though. I just find the double-negative confusing, but I can't English sometimes so maybe this is a me-issue.

@justinmk any thoughts on clarity here?